tests/unit_tests/CMakeLists.txt

```diff
@@ -18,9 +18,13 @@ target_compile_options(
 target_include_directories(${oturb_unit_test_exe_name} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
 target_include_directories(${oturb_unit_test_exe_name} PRIVATE ${PROJECT_BINARY_DIR})
 
+if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "9.0")
```

Comment:
Is this specific to gcc?

Reply:
To my knowledge, yes.  There may be similar issues for other compilers, but I haven't tested them.  I should add an extra check to only add this flag for GCC.